lib/client/client.js

```diff
@@ -23,8 +23,8 @@ const run = (buildHash, options) => {
   const { replace } = require('./hmr');
   const { error, info, warn } = require('./log')();
 
-  const protocol = secure ? 'wss' : 'ws';
-  const socket = new ClientSocket(client, `${protocol}://${client.address || address}/wps`);
+  const serverProtocol = secure ? 'wss' : 'ws';
```

Comment:
```suggestion
  const protocol = secure ? 'wss' : 'ws';
```

Reply:
I changed the name because I thought it would be clearer to specify that the original protocol refers to the protocol scheme served by the server, not necessarily the one requested by the client. But I'm not tied to this, happy to revert!